spec/models/manageiq/providers/ibm_terraform/configuration_manager/refresher_spec.rb

```diff
@@ -61,7 +61,6 @@ def assert_specific_configuration_profile
         :name              => "LAMP stack deployment on AWS",
         :description       => "LAMP - A fully-integrated environment for full stack PHP web development.",
         :target_platform   => "Amazon EC2",
-        :supports_console? => true,
```

Comment:
could you add to both of these?

```ruby
expect(configuration_profile.supports?(:native_console)).eq (true)
```

Reply:
This is the refresher spec where `supports?(:native_console)` is a constant not set by the refresher so it doesn't really make sense to test for it here